apis/Google.Cloud.Tasks.V2Beta2/Google.Cloud.Tasks.V2Beta2.Tests/CloudTasksClientTest.g.cs

```diff
@@ -637,8 +641,8 @@ public void GetIamPolicy()
             mockGrpcClient.Setup(x => x.GetIamPolicyAsync(expectedRequest, It.IsAny<CallOptions>()))
                 .Returns(new Grpc.Core.AsyncUnaryCall<Policy>(Task.FromResult(expectedResponse), null, null, null, null));
             CloudTasksClient client = new CloudTasksClientImpl(mockGrpcClient.Object, null);
-            string formattedResource = new QueueName("[PROJECT]", "[LOCATION]", "[QUEUE]").ToString();
-            Policy response = await client.GetIamPolicyAsync(formattedResource);
+            QueueName resource = new QueueName("[PROJECT]", "[LOCATION]", "[QUEUE]");
```

Comment:
... but we're creating a queue name resource to pass in.

Reply:
Oh yes, not sure why that's happening. Will fix in a new PR.